sources/packages/web/src/assets/css/global-style-variables.scss

```diff
@@ -61,6 +61,8 @@ $badge-background-color: white;
 /** font weight bold**/
 $font-weight-bold: bold;
 
+$font-weight-bolder: bolder;
```

Comment:
why bolder? bold doesn't work??

Reply:
It was created for icons. Will remove it as per my previous comment.